source/docs/guides/settlement.md

```diff
@@ -0,0 +1,179 @@
+---
```

Comment:
Should url be settlement_report to keep it consistent with other guides and not as a guide on how settlement works?

Reply:
Good suggestion, done.